index.html

```diff
@@ -17,7 +17,8 @@
             <video
                 id="vuplay-video"
                 poster="vuplay_poster.png"
-                controls="controls"
+                controls
+                muted
```

Comment:
muted?

Reply:
Yes, I was annoyed during the testing to start the video with the sound and like that the autostart works.